scripts/ec2/describe_addresses.coffee

```diff
@@ -0,0 +1,26 @@
+# Description:
+#   List elastic IP's
+#
+# Commands:
+#   hubot ec2 describe addresses
+
+aws = require('../../aws').aws()
+ec2 = new aws.EC2()
+
+module.exports = (robot) ->
+  robot.respond /ec2 describe addresses$/i, (msg) ->
+    msg.send "Fetching ..."
+    ec2.describeAddresses {}, (err,response) ->
+      if err
+        msg.send err
+      else
```

Comment:
I want to avoid this error. 
This error is reproduced when there is no allocated addresses. 

```
hubot> hubot ec2 describe addresses
Fetching ...
hubot> 
[Sat Aug 20 2016 12:46:21 GMT+0900 (JST)] ERROR TypeError: Object.keys called on non-object
  at Function.keys (native)
  at Response.<anonymous> (/mtburn/dev/js/hubot/github/hubot-aws/scripts/ec2/describe_addresses.coffee:17:9, <js>:13:25)
```